src/main/java/Duke.java

```diff
@@ -1,10 +1,66 @@
+import java.util.Scanner;
+
 public class Duke {
+
+    public static final String HORIZONTAL_LINES = "____________________________________________________________\n";
+    public static final String INDENTATIONS = "    ";
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        System.out.println(INDENTATIONS + HORIZONTAL_LINES +
+                INDENTATIONS + "Hello! I'm Duke\n" +
+                INDENTATIONS + "What can I do for you?\n" +
+                INDENTATIONS + HORIZONTAL_LINES);
+        Scanner in = new Scanner(System.in);
+        String readLine;
+        Task[] tasks = new Task[100];
+        int taskIndex = 0;
+        while(true) {
+            readLine = in.nextLine();
+            if(readLine.equals("bye")) {
+                System.out.println(INDENTATIONS + HORIZONTAL_LINES +
+                        INDENTATIONS + "Bye. Hope to see you again soon!\n" +
+                        INDENTATIONS + HORIZONTAL_LINES);
+                break;
+            } else if (readLine.equals("list")) {
+                // list out the tasks and status in todo
+                System.out.println(INDENTATIONS + HORIZONTAL_LINES +
+                        INDENTATIONS + "Here are the tasks in your list: ");
+
+                for (int i = 0; i < taskIndex; ++i) {
+                    int taskNumber = i+1;
+                    System.out.println(INDENTATIONS + taskNumber + "." +
+                            "[" + tasks[i].getStatusIcon() + "] " + tasks[i].getTaskDescription());
+                }
+
+                System.out.println(INDENTATIONS + HORIZONTAL_LINES);
+
+            } else if (readLine.startsWith("mark")) {
+                String toMark = readLine.substring(5);
+                toMark = toMark.trim();
+                int indexToMark = Integer.parseInt(toMark) - 1;
+                tasks[indexToMark].setDone(true);
+                System.out.println(INDENTATIONS + "Nice! I've marked this task as done:\n" +
+                        INDENTATIONS + "  [X] " + tasks[indexToMark].getTaskDescription() + '\n' +
+                        INDENTATIONS + HORIZONTAL_LINES);
```

Comment:
I like how you ensured that the lines to be printed out are wrapped when appropriate, making the code easier to read.